src/main/java/seedu/address/model/loan/Nric.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.model.loan;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Class that stores the NRIC of a person.
+ */
+public class Nric {
+
+    public static final String MESSAGE_NRIC_CONSTRAINTS =
+            "Nric should be singapore issued. It may be blank. ";
```

Comment:
Please capitalise "Singapore".

Reply:
"NRIC" should be capitalized as well, since it's a proper acronym in a message to the user.